tests/appsec/iast/sink/test_hsts_missing_header.py

```diff
@@ -0,0 +1,50 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, coverage, released, missing_feature, weblog
+from .._test_iast_fixtures import SinkFixture
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+@released(java="1.20.0")
```

Comment:
You must use only the manifest file to declare this

Reply:
applied